docs/Feature Documentation/index.md

```diff
@@ -0,0 +1,10 @@
+## SDL Core Feature Documentation
+
+Here you will find guides on how to use different features of the SDL Core application.
```

Comment:
Is "application" the best word here?  I'm wondering if just "...features of SDL Core." is better, to avoid confusion with apps using one of the proxy SDKs.

Reply:
SDL Core as the proper noun is probably better, that is how we refer to it throughout most of the guides.

Changed in https://github.com/smartdevicelink/sdl_core_guides/pull/71/commits/d473987b5831c7722aeb714092231c8ced049b76